### PR TITLE
cards for ALP linear and chiral models

### PR DESCRIPTION
**Pull request by xdlyu**
Cards for ZZ production using the ALP_linear model and the ZH production using the ALP_chiral model

### CODE REVIEW

**Comment by sihyunjeon** [comment]
@agrohsje I don't see any issues with these cards

**Comment by agrohsje** [comment]
Hi @xlyu what is the purpose of the PROC_ALP_linear_UFO_0.tar that I was asked to copy. I copied the models ending with tar.gz, the ones that are here loaded. Please let me know about  PROC_ALP_linear_UFO_0.tar/PROC_ALP_chiral_UFO_0.tar .

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ALP_linear_chiral/ZH_ALP_chiral/ZH_ALP_chiral_proc_card.dat [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import model ALP_chiral_UFO
+define l+ = e+ mu+
```
Comment (agrohsje) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ALP_linear_chiral/ZH_ALP_chiral/ZH_ALP_chiral_proc_card.dat:2]: What is the reason not to have Z -> tau tau, where tau can also give e, mu ?

Comment (xdlyu) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ALP_linear_chiral/ZH_ALP_chiral/ZH_ALP_chiral_proc_card.dat]: I do not know if I understand your first question correctly. As the author (Jorge Troconiz) said, the two models correspond to two EFT implementations: linear (simpler, contains ZZ and W+W- production) and chiral (more parameters, contains ZH production). And the models are uploaded to the CMS generator web repository. (https://cms-project-generators.web.cern.ch/cms-project-generators/)

For the Z -> tau tau, there is no objection to include taus as well.  I just committed a new version to add it.

**Comment by agrohsje** [comment]
Hi @xdlyu. I know,  I copied the models to the CMS generator web repo. My question was about two additional files, e.g. PROC_ALP_linear_UFO_0.tar which Jorge provided. I am not sure about the purpose of this and didn't copy it.

**Comment by xdlyu** [comment]
> Hi @xdlyu. I know, I copied the models to the CMS generator web repo. My question was about two additional files, e.g. PROC_ALP_linear_UFO_0.tar which Jorge provided. I am not sure about the purpose of this and didn't copy it.

I see, the corresponding process/MadGraph setup are called PROC_ALP_*_UFO_0.tar. So only the 2 files with tar.gz in the name are for the models.

**Comment by agrohsje** [comment]
Perfect. So all good.

**Pull request merged**